build.sbt

```diff
@@ -89,7 +90,7 @@ lazy val natchezDatadog = projectMatrix
       libraryDependencies ++= Seq(
         "org.http4s" %% "http4s-dsl" % http4sStableVersion,
         "org.http4s" %% "http4s-circe" % http4sStableVersion,
-        "org.http4s" %% "http4s-client" % http4sStableVersion
+        "org.http4s" %% "http4s-client" % blazeStableVersion
```

Comment:
According to the README here: https://github.com/http4s/blaze#standalone-modules

Only `http4s-blaze-*` packages have been moved out from the main repo. As far as I understands the `http4s-client` package is implementation agnostic, and follows the main version.

Reply:
returned to the original version, thanks!